doc/pages/faq.asciidoc

```diff
@@ -34,13 +34,29 @@ This variable should be set by the terminal emulator, and not overridden with
 an arbitrary value, otherwise it might interfere with general UI rendering on
 the terminal's window.
 
+Currently, Kakoune relies on "palette switching": redefining the standard
+256-color palette for true 24-bit RGB color support.
+If you're using a tool that doesn't support "palette switching", colors will
+be slightly off: they are being rounded down to values supported by the
+regular color palette by Kakoune.
+
+== Why are colors unreadable in `konsole`/`QTermWidget` or `st`?
+
+Even if the `TERM` and `COLORTERM` environment variables are set correctly, most
+colorschemes look off. That's because these terminal emulators don't support
+"palette switching" (see above).
+For further information, see https://github.com/mawww/kakoune/issues/1807[#1807].
```

Comment:
The future of Kakoune colour support is almost certainly the `builtin-terminal-ui` branch, which ignores `TERM` and assumes it's running inside a modern terminal.

Reply:
When that branch is merged, we'll have to edit the FAQ anyway. But until then I'd like to spare others the troubles of trying to figure out how to get colors working in konsole.
But you're right, reading the linked issue does not really help. I'll edit the PR.